Makefile

```diff
@@ -26,7 +26,7 @@ endif
 
 BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
 SHORT_SHA=$(shell git rev-parse --short HEAD)
-VERSION?=${BRANCH}-${SHORT_SHA}
+VERSION?=$(shell ./hack/git-version.sh)
```

Comment:
What have we talked about this version and its interplay with other tools? IIRC you mentioned somethinig about docker images having an issue with full sem-ver tagging due to some special characters? Could we keep the VERSION as it was and get this merged/into proper PR review.